### ISSUE TITLE
Keeping in touch

### ISSUE THREAD

**Issue by lebuhn** [bwainwri/brooke-gretchen#2]
My email is lebuhn@sfsu.edu and phone is 415-847-1716.  Let's set up a time to talk about your work.  I'd love to learn more.

**Comment by bwainwri** [comment]
Yes! Let's! My email is bwainwright@ucdavis.edu and number is 510-599-5068. I'll reach out soon! 

I just learned you can make custom labels under the gear icon under labels and "edit labels". So fun.
